dotcom-rendering/src/components/Carousel.importable.tsx

```diff
@@ -526,17 +531,217 @@ const HeaderAndNav = ({
 	</div>
 );
 
-const decideCarouselColours = (
-	props: { format: ArticleFormat } | { palette: DCRContainerPalette },
-): {
+const Header = ({
+	heading,
+	trails,
+	carouselColours,
+	index,
+	goToIndex,
+	prev,
+	next,
+	arrowName,
+	isCuratedContent,
+	isVideoContainer,
+	hasPageSkin,
+}: {
+	heading: string;
+	trails: TrailType[];
+	carouselColours: CarouselColours;
+	index: number;
+	goToIndex: (newIndex: number) => void;
+	prev: () => void;
+	next: () => void;
+	arrowName: string;
+	isCuratedContent: boolean;
+	isVideoContainer: boolean;
+	hasPageSkin: boolean;
+}) => {
+	const header = (
+		<div css={headerRowStyles}>
+			<HeaderAndNav
+				heading={heading}
+				trails={trails}
+				titleHighlightColour={carouselColours.titleHighlightColour}
+				titleColour={carouselColours.titleColour}
+				activeDotColour={carouselColours.activeDotColour}
+				index={index}
+				isCuratedContent={isCuratedContent}
+				goToIndex={goToIndex}
+			/>
+			<Hide when="below" breakpoint="desktop">
+				<button
+					type="button"
+					onClick={prev}
+					aria-label="Move carousel backwards"
+					css={[
+						buttonStyle(
+							carouselColours.arrowColour,
+							carouselColours.arrowBackgroundColour,
+							carouselColours.arrowBackgroundHoverColour,
+						),
+						prevButtonStyle(
+							index,
+							carouselColours.arrowColour,
+							carouselColours.arrowBackgroundColour,
+							carouselColours.arrowBackgroundHoverColour,
+						),
+					]}
+					data-link-name={getDataLinkNameCarouselButton(
+						'prev',
+						arrowName,
+						isVideoContainer,
+					)}
+				>
+					<SvgChevronLeftSingle />
+				</button>
+				<button
+					type="button"
+					onClick={next}
+					aria-label="Move carousel forwards"
+					css={[
+						buttonStyle(
+							carouselColours.arrowColour,
+							carouselColours.arrowBackgroundColour,
+							carouselColours.arrowBackgroundHoverColour,
+						),
+						nextButtonStyle(
+							index,
+							trails.length,
+							carouselColours.arrowColour,
+							carouselColours.arrowBackgroundColour,
+							carouselColours.arrowBackgroundHoverColour,
+						),
+					]}
+					data-link-name={getDataLinkNameCarouselButton(
+						'next',
+						arrowName,
+						isVideoContainer,
+					)}
+				>
+					<SvgChevronRightSingle />
+				</button>
+			</Hide>
+		</div>
+	);
+	/**
+	 * When there is a page skin we constrain to a desktop layout
+	 * So don't hide the header past leftcol as normal
+	 */
+	if (hasPageSkin) {
```

Comment:
When there is a page skin we constrain to a desktop layout. So don't hide the header past leftcol as normal